fix-point/app/Main.hs

```diff
@@ -10,19 +11,26 @@ import CopyFiles (copyAll)
 import Rename
 
 main :: IO ()
-main = test =<< execParser opts
+main = bootstrap =<< execParser opts
   where
     opts = info (parseOptions <**> helper)
       (  fullDesc
-      <> progDesc "Test parsing of CLI's"
-      <> header "Test for optparse-applicative")
+      <> progDesc "Refactors with project, prefix and sourcepath with three-layer"
+      <> header "Content refactoring")
 
-test :: Options -> IO ()
-test opts = case opts of
-    (Options proj (Just pref) (Just sd)) -> copyAll sd proj pref
-    (Options proj Nothing (Just sd))     -> copyAll sd proj proj
-    (Options proj (Just pref) Nothing)   -> copyAll "three-layer" proj pref
-    (Options proj Nothing Nothing)       -> copyAll "three-layer" proj proj
+bootstrap :: Options -> IO ()
+bootstrap (Options project pref source) = do
+    let prefix = case pref of
+                     (Just s) -> s
+                     Nothing  -> (f project)
+    let sourceDir = case source of
+                        (Just s) -> s
+                        Nothing  -> "three-layer"
```

Comment:
And here you can use `fromMaybe` function:

* https://hackage.haskell.org/package/base-4.11.1.0/docs/Data-Maybe.html#v:fromMaybe